index.js

```diff
@@ -239,11 +250,8 @@
             this.editor.off('click', this.editorClickProxy);
 
             document.body.removeEventListener('click', this.bodyClickProxy);
-
-            var editorWindow = this.editor.getWin();
-            if (editorWindow) { //is the editor still visible?
-                editorWindow.removeEventListener('scroll', this.rteScroll);
-            }
+            document.removeEventListener('scroll', this.rteScroll);
+            document.removeEventListener('resize', this.rteResize);
```

Comment:
Shouldn't it be `window.removeEventListener`?

Reply:
Sure